rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -697,8 +704,11 @@ def _decode_transaction_hashes(
 
         refresh_balances = False
         total_transactions = len(tx_hashes)
+        log.debug(f'Started logic to decode {total_transactions} from {self.evm_inquirer.chain_id}')  # noqa: E501
         for tx_index, tx_hash in enumerate(tx_hashes):
+            log.debug(f'Decoding logic started for {tx_hash.hex()}({self.evm_inquirer.chain_name})')  # noqa: E501
             if send_ws_notifications and tx_index % 10 == 0:
+                log.debug(f'Processed {tx_index} out of {total_transactions} from {self.evm_inquirer.chain_id}')  # noqa: E501
```

Comment:
Please be consistent in the logs

```suggestion
                log.debug(f'Processed {tx_index} out of {total_transactions} transactions from {self.evm_inquirer.chain_name}')  # noqa: E501
```